src/api/app/mixins/webui/manage_relationships.rb

```diff
@@ -1,12 +1,6 @@
 module Webui::ManageRelationships
-  def redirect_after_save(what)
-    if switch_to_webui2
-      redirect_to(users_path)
-    elsif what == :person
-      redirect_to(add_path(:add_person))
-    else
-      redirect_to(add_path(:add_group))
-    end
+  def redirect_after_save
```

Comment:
nitpick: that function is only used once and a one liner

Reply:
done